contrib/babelfishpg_tsql/src/pltsql_bulkcopy.c

```diff
@@ -102,6 +102,8 @@ BulkCopy(BulkCopyStmt *stmt, uint64 *processed)
 	{
 		if (!stmt->cstate)
 			stmt->cstate = BeginBulkCopy(rel, attnums);
+		else
+			stmt->cstate->rel = rel;
```

Comment:
We should look into the possibility of adding or enhancing test case here. Also, we should see if certain additional validation checks can be added to make sure DSs are in valid state.